src/main/java/com/salesforce/storm/spout/dynamic/VirtualSpout.java

```diff
@@ -256,16 +257,12 @@ public void close() {
      */
     @Override
     public Message nextTuple() {
-        // Start total Time timer.
-        getMetricsRecorder().startTimer(getClass(), "nextTuple.entireMethod");
```

Comment:
Not sure that I'm crazy about removing these, I think we should leave them in here honestly.